source/MapPanel.cpp

```diff
@@ -1280,9 +1280,10 @@ void MapPanel::DrawEscorts()
 				{
 					// Starting at 7.5 degrees to intentionally mis-align with mission pointers.
 					Angle angle = Angle(7.5f + 45.f * i);
-					Point from = pos + angle.Unit() * OUTER;
-					Point to = from + angle.Unit() * 4.f;
-					LineShader::Draw(from, to, 2.f, active);
+					constexpr float width = 1.6f;
+					Point from = pos + angle.Unit() * (OUTER + width);
+					Point to = from + angle.Unit() * (4.f - width);
+					LineShader::Draw(from, to, width, active);
```

Comment:
```suggestion
					constexpr float WIDTH = 1.6f;
					Point from = pos + angle.Unit() * (OUTER + WIDTH);
					Point to = from + angle.Unit() * (4.f - WIDTH);
					LineShader::Draw(from, to, WIDTH, active);
```
Consider making it a more meaningful name and move to the other consts in mappanel.h

Reply:
The other constants are in mappanel.h because they are used elsewhere in the codebase and have to account for being ODR'd, but in this case the scope of the constant is small enough that it is more readable when in the same scope.